coala_quickstart/generation/FileGlobs.py

```diff
@@ -16,6 +16,8 @@ def get_project_files(log_printer, printer, project_dir):
         A ``LogPrinter`` object.
     :param printer:
         A ``ConsolePrinter`` object.
+    :param noninteractive:
+        The running mode for coala-quickstart.
```

Comment:
Maybe something like `Whether coala-quickstart is in non-interactive mode.` would be a better description? :)